Source/ListParametersCommand.cpp

```diff
@@ -41,8 +41,8 @@ void ListParametersCommand::execute() {
 
         // print the parameter's values
         std::cout << indent << "Values:  ";
-
-        if (auto valueStrings = param->getAllValueStrings(); !valueStrings.isEmpty()) {
+        auto valueStrings = param->getNumSteps() < 50 ? param->getAllValueStrings() : juce::StringArray();
```

Comment:
Can you explain this?

Reply:
See comment below (apparently it wasn't visible to you because it was part of an unsubmitted "review")